packages/fetch-http-handler/src/fetch-http-handler.ts

```diff
@@ -9,7 +9,7 @@ declare let AbortController: any;
 /**
  * Represents the http options that can be passed to a browser http client.
  */
-export interface BrowserHttpOptions {
+export interface FetchHttpHandlerOptions {
```

Comment:
Are we renaming this just to match the file?

Reply:
> Are we renaming this just to match the file?

To match the class name where options interface is passed to.